integrations/facebook-pixel/test/index.test.js

```diff
@@ -86,6 +90,20 @@ describe('Facebook Pixel', function() {
         analytics.assert(window.fbq instanceof Function);
       });
 
+      before(function() {
```

Comment:
I would omit these `before` and `after` blocks and just explicitly toggle this option before and after the test code runs. So like this:

```
      it('should call set autoConfig if option disableAutoConfig is enabled', function () {
options. automaticConfiguration = false
        analytics.stub(window, 'fbq');
        analytics.initialize();
        analytics.called(window.fbq, 'set', 'autoConfig', false, options.pixelId);
options.automaticConfiguration = true;
      });
```

Reply:
Unless you think there would be a need for future tests, in which case, I would use `beforeEach` and `afterEach`.

Reply:
Ahh, doing it in the test code doesn't work since a.js is initialized before-hand.